solutions/landing-zone-v2/README.md

```diff
@@ -0,0 +1,585 @@
+# KRM Landing Zone v2
+
+This solution uses [Config Controller](https://cloud.google.com/anthos-config-management/docs/concepts/config-controller-overview) to deploy and manage the GCP infrastructure. 
+
+Config Controller is a hosted service to provision and orchestrate Anthos, GKE, Kubernetes and Google Cloud resources. It offers an API endpoint that can provision, actuates, and orchestrates Google Cloud resources as part of Anthos Config Management.
+
+![img](img/ACM.png)
```

Comment:
The images in this README are not loading. The issue is that the filenames are case-sensitive so we need to match the uppercase extension which is `PNG` and not `png`. Please check/fix all other links.

Reply:
+1